be/src/storage/persistent_index.h

```diff
@@ -21,6 +21,13 @@ class Schema;
 class Column;
 } // namespace vectorized
 
+// Add version for persistent index file to support future upgrade compatibility
+// There is only one version for now
+enum IndexFileVersion {
+    UNKNOWN = 0,
+    VERSION_1,
```

Comment:
```suggestion
    PERSISTENT_INDEX_VERSION_1,
```

version_1 is too common

Reply:
> version_1 is too common

done, changed